src/Microsoft.AspNetCore.HttpsPolicy/HttpsRedirectionMiddleware.cs

```diff
@@ -94,28 +97,41 @@ public Task Invoke(HttpContext context)
             context.Response.StatusCode = _statusCode;
             context.Response.Headers[HeaderNames.Location] = redirectUrl;
 
+            _logger.LogDebug($"Redirecting to https.");
```

Comment:
Log the host string here?

Reply:
Why? It's not part of the decision.

Reply:
Yeah a log mentioning which host/port we are redirecting to.

Reply:
Use the LoggerMessage.Define pattern for logs in our product code.